agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/OpenTelemetryConfigurer.java

```diff
@@ -116,6 +118,14 @@ public void configure(SdkTracerProviderBuilder tracerProvider, ConfigProperties
               configuration.preview.instrumentationKeyOverrides));
     }
 
+    // Add quick pulse
+    tracerProvider.addSpanProcessor(
+        new LiveMetricsSpanProcessor(
+            telemetryClient.getAadAuthentication(),
+            telemetryClient.getRoleName(),
+            telemetryClient.getInstrumentationKey(),
+            telemetryClient.getRoleInstance(),
+            ConnectionString.parse(configuration.connectionString).getLiveEndpoint().toString()));
```

Comment:
let's not add the span processor to the agent in this PR, 2.x bridged data won't flow through this, and we already flush the BatchSpanProcessor very fast in the agent and batch at a lower layer

Reply:
Adding the LiveMetricsSpanProcessor is the only way we can enable quickpulse to the agent (since we removed the old code). Do you recommend any other alternative?

Reply:
I'm not clear what you mean, `QuickPulseDataCollector.INSTANCE.add(telemetryItem)` is still being called from TelemetryClient so this isn't needed (and is probably causing duplicate live metrics?)